unicorn/CMakeLists.txt

```diff
@@ -1,15 +1,37 @@
 INCLUDE_DIRECTORIES(
     ${PROJECT_SOURCE_DIR}/unicorn/include)
 
+# Headers are not required but can be usefull in some cases,
+# for example in some IDEs it's required to add files to project.
+SET(HEADERS
```

Comment:
Please remove this. I want to reduce number of unnecessary code to the minimum to make everything compact.